content/blog/2021/07/camel-jbang/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Camel + JBang: bringing the world of Java scripting to Camel"
+date: 2021-07-21
```

Comment:
You may delay a couple of days as we just published another Camel K blog: https://camel.apache.org/blog/2021/07/camel-k-properties-refactoring/

Reply:
+1. Let's move it to next week

Reply:
We didn't configure Hugo to build future posts, so no checks will be performed as there will be no HTML file generated to check, so be aware of that.

Reply:
@zregvart thanks for the heads up! For now I'll set it as July 20th, but I added a checklist to adjust the date before merge.